conda-forge.yml

```diff
@@ -3,3 +3,7 @@ build_platform:
 conda_forge_output_validation: true
 provider:
   win: azure
+remote_ci_setup:
+  - conda-forge-ci-setup=3
+  - conda-forge/lief_dev::py-lief>=0.12.0
```

Comment:
```suggestion
  - conda-forge/label/lief_dev::py-lief>=0.12.0
```

Reply:
I knew i had something wrong! Thanks. Hopefully i'll be able to use this to write some documentation.